Control/inverted_pendulum/inverted_pendulum_lqr_control.py

```diff
@@ -0,0 +1,185 @@
+"""
```

Comment:
Could you add a unit test code for this simulation?

Reply:
Added in the last commit.
I also enable the unit test of inverted_pendulum_mpc_control.py by removing check 
cvxpy in sys.module (like test_model_predictive_speed_and_steer_control.py)